common/component_templates/00_utilities_shields.txt

```diff
@@ -547,6 +604,7 @@ utility_component_template = {
 		category = ship_components
 		cost = {
 			alloys = @shield_s_t7_cost
+			sr_zro = 0.111
```

Comment:
I'll not show every further resource you get the deal, every strategic just got added back in as part of the update.